package exception;

public class ArrayHandlingExcep {
	public static void main(String[] args) 
	{  
		String arr[] = {"Aparna","Meenu","Nithya","Sajitha"};   
	 	try
	 	{
		      System.out.println(arr[4]); 
	 	}
	 	catch(Exception e)
	 	{
	 		System.out.println(e);
	 	}
		          
		    }  
		  
		    }  


